### ISSUE TITLE
please make web ui for Oauth authorize code request & response value field

### ISSUE THREAD

**Issue by bugbounted** [Pokesi/tiktok2youtube#3]
Hi [Pokesi](https://github.com/Pokesi),

please make web ui for Oauth authorize code request because I'm make changes for made dockerized version on my fork & deploy it to railway , but after deployed request me to visit generated oauth link & input value & in railway deploy log console can't type anything (answer of oauth link) . please make changes on your youtube upload section code to send oath link & ask response field to web or any possible way to upload video to youtube without oauth authorize webbase needed.

waiting for your response
finest regards

**Comment by Pokesi** [comment]
Hey, the google OAuth code isn't handled by this code, it's handled by a google website that should pop up under a non-docker deployment.

TL;DR You can't run this in docker.
